packages/gatsby-plugin-twitter/src/gatsby-browser.js

```diff
@@ -27,10 +27,15 @@ const injectTwitterScript = () => {
 
 let injectedTwitterScript = false
 
+const embedClasses = `.twitter-tweet,
+  .twitter-timeline,
+  .twitter-follow-button,
+  .twitter-share-button`
```

Comment:
I wonder if 
```js
const embedClasses = [
  `.twitter-tweet`,
  `.twitter-timeline`,
  `.twitter-follow-button`,
  `.twitter-share-button`
].join(`,`)
```
would be more readable?

On initial check I didn't notice `.twitter-tweet` being still there because it wasn't aligned with rest of selectors

Reply:
Sure! Fixed that.